rotkehlchen/chain/evm/decoding/base.py

```diff
@@ -171,13 +171,13 @@ def decode_erc20_721_transfer(
         event_type, event_subtype, location_label, address, counterparty, verb = direction_result
         counterparty_or_address = counterparty or address
         amount_raw_or_token_id = int.from_bytes(tx_log.data)
-        if token.token_kind == EvmTokenKind.ERC20:
+        if token.token_kind == TokenKind.ERC20:
```

Comment:
Isn't it better to use match statement here to make sure all TokenKinds are handled?